docker-dev/docker-compose.yml

```
@@ -0,0 +1,69 @@
+version: '3'
+
+services:
+  vs-agent:
+    image: io2060/vs-agent:dev
+    restart: always
+    networks:
+      - chatbot
+    ports:
+      - 3000:3000
+      - 3001:3001
+    environment:
+      - AGENT_PUBLIC_DID=did:webvh:your-ngrok-domain.ngrok-free.app
+      - AGENT_INVITATION_IMAGE_URL=https://your-ngrok-domain.ngrok-free.app/logo.png
+      - AGENT_LABEL=Concieragent
+      - USE_CORS=true
+      - EVENTS_BASE_URL=https://your-ngrok-domain.ngrok-free.app
```

I recommended setting the following environment variables in the VS Agent 
```suggestion
      - EVENTS_BASE_URL=https://your-ngrok-domain.ngrok-free.app
      - POSTGRES_HOST=postgres
      - POSTGRES_USER=${POSTGRES_USER:-postgres}
      - POSTGRES_PASSWORD=${POSTGRES_PASSWORD}
```